macro/src/lib.rs

```diff
@@ -41,7 +41,10 @@ use syn::parse_macro_input;
 pub fn bridge(args: TokenStream, input: TokenStream) -> TokenStream {
     let _ = syntax::error::ERRORS;
 
-    let namespace = parse_macro_input!(args as Namespace);
+    let namespace = match Namespace::parse_bridge_attr_namespace.parse(args) {
+        Ok(ns) => ns,
+        Err(err) => return err.to_compile_error().into(),
+    };
```

Comment:
Wrote https://github.com/dtolnay/syn/pull/896 to add a case to parse_macro_input! for this usage